test/KeyTest.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import os
+sys.path.append(os.path.abspath(os.path.dirname(__file__) + '/' + '../'))
+from lib import KeyGenUtils
+
+
+# Demo
+# By System
+key = KeyGenUtils.get_key_by_system()
+print(KeyGenUtils.__get_mac__() + "   " + key)
+print(KeyGenUtils.verify_key_by_system(key))
+
+# By Username
+key = KeyGenUtils.get_key_by_username()
+print(KeyGenUtils.__get_current_user__() + "   " + key)
```

Comment:
_get_current_user_() is custom function right, use camelCasing here, also for all the above functions too. I think these are the last changes that you need to revise.